### PR TITLE
Added piplock with fast api

### PR DESCRIPTION
**Pull request by saisankargochhayat**
Initialized the project with Pipenv
Added an empty cronjob folder to add cronjob code.

### CODE REVIEW

**Comment by saisankargochhayat** [comment]
Please merge this, we can use Pipenv for our dependencies.

File: notebooks/.ipynb_checkpoints/runQiskit-checkpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,113 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 6,
```
Comment (rsquare2010) [inline notebooks/.ipynb_checkpoints/runQiskit-checkpoint.ipynb]: Hey, what does the Jupiter notebook stuff do?

Comment (varshithauppula96) [reply notebooks/.ipynb_checkpoints/runQiskit-checkpoint.ipynb]: It's the sample Qiskit circuit we have implemented to check the output generated.

Comment (saisankargochhayat) [reply notebooks/.ipynb_checkpoints/runQiskit-checkpoint.ipynb]: Removed the checkpoint files and added hypercorn to dependencies.

**Review by rsquare2010 — APPROVED** [review]

**Comment by rsquare2010** [comment]
LGTM, will wait on others to merge..

**Review by srihitha97 — APPROVED** [review]

**Review by varshithauppula96 — APPROVED** [review]

**Pull request merged**